rotary_utils/repair.py

```diff
@@ -55,12 +46,6 @@ def main():
 
     os.makedirs(args.output_dir, exist_ok=True)
 
-    # Start log file in the output dir
-    file_handler = logging.FileHandler(filename=os.path.join(args.output_dir, 'repaired.log'), mode='w')
```

Comment:
For now, I remove the log file generated by `rotary-utils repair` to simplify the code. Instead, logging is only printed to screen. We can revisit potentially re-adding the auto-generated log file later, but it might take some re-thinking of how logging is set up in `rotary-utils`.

Reply:
The main issue that makes it complicated to keep the auto-generated log file is that the file logger (for the log file) currently needs to be started in `main` of `repair.py`, rather than in `rotary_utils.py`, where the console logger is set up. The reason why the file logger needs to be started in `main` of `repair.py` is that log file can only be set up after the output directory for the run is created by `main`. Needing to set up the file logger in `repair.py` causes duplicate code (e.g., logging formatter) to be needed between `rotary_utils.py` and `repair.py`. It also makes it a bit complicated to incorporate logging from other modules like `utils.py`.

I imagine that there might be a more elegant way to set up logging in rotary-utils (e.g., move some or all of the `main` function of `repair.py` to `rotary_utils.py`, as I hint at elsewhere in this review). Let me know if you have any thoughts. Otherwise, I figure I can come back and address this log file issue later, given that it's not a critical issue, or we might decide that it's not a needed feature for the MVP.

Reply:
Usually, when logging into the file, I pass the log path as an optional CLI parameter. I also control the log level via a CLI parameter.

Reply:
@LeeBergstrand Thanks for your feedback. I hope to pass the log path as an optional CLI parameter for other utilities that don't make a dedicated output directory. For the repair utility, because it already makes a dedicated output dir, I figure it could help the user to just automatically write a log file to the newly made output dir. The `-v` param already controls log level for both the console logger and file logger. How does that approach sound?

Reply:
@LeeBergstrand I just created #2 that addresses issues with logging. Now, the log file is written automatically in the output dir, restoring the functionality I removed. In addition, I went ahead and added a parent setting that lets a user specify a log file at a custom file path. This parent setting can be accessed for any module. Look forward to your review of #2.